docs/templates/about.html

```diff
@@ -0,0 +1,98 @@
+        </header>
+        <p class="general-margin center-border normal-font">
+            Developers listed on this website are directly or indirectly responsible for today's programming world by their great influence. These programmers pioneered the computing world and changed the way we access information today.
+            <br><br>
+            By definition, a computer programmer is an inventor of computer programs. Okay, that much is evident but he is also the person who then alters these programs to suit the needs. In a way, all programmers contribute to our ever evolving society which is becoming heavily dependent on technology.
+            It may be in a small measure or it can be in a major way. Irrespective of in which manner they added value to the society they are creative people and creativity is what shapes the world. So these are bold personnel who dared to spend their lives and time to benefit humankind.
+        </p>
+        <div class="general-margin center-text contain-content inherit-size">
+            <p class="normal-font center-text small-text no-margin contain-content">Please contact us per email for any further questuions about GI website</p>
+            <a class="mail small-text overflow" href="../index.html">get-inspired@example.org</a>
+        </div>
+    </section>
+    <!-- Section 2 -->
+    <section class="section-about-2">
+        <h2 class="contain-content">Get Inspired 2022 logo</h2>
+        <div class="guide-bar"></div>
+        <p class="normal-font center-text small-text no-margin contain-content">The logo of GI was made in 2022 through the website logaster.com</p>
+        <img class="general-margin" src="../sprites/Get-inspired-brand-logo.png" alt="Get Inspired logo">
+    </section>
+    <!-- Section 3 -->
+    <section class="section-about-3">
+        <h2>Learn more about influential developers.</h2>
+        <div class="guide-bar"></div>
+        <p class="normal-font center-text small-text no-margin">Visit this references if you want to learn more about great developers.</p>
+        <div>
+            <a class="a1" href="https://thedeveloperstory.com/2021/06/09/11-most-influential-greatest-programmers-of-all-time/">
+                <p>Most influential & greatest programmers of all time</p>
+            </a>
+            <a class="a2" href="https://www.csestack.org/most-influential-computer-programmers/">
+                <p>World Most Influential Computer Programmers in History!</p>
+            </a>  
+        </div>  
+    </section>
+    <!-- Section 4 -->
+    <section class="section-4">
+      <h2>Partners</h2>
+      <div class="guide-bar"></div>
+      <div class="brands">
+        <img src="../sprites/part1.svg" alt="">
+        <img src="../sprites/part2.svg" alt="">
+        <img src="../sprites/part3.svg" alt="">
+        <img src="../sprites/part4.svg" alt="">
+        <img src="../sprites/part5.svg" alt="">
+        <img src="../sprites/part1.svg" alt="">
```

Comment:
- [ ] Please add alt text to all images this is important so when browsers are unable to render a page properly a text is displayed 👍🏿